StoreSupplier

```diff
@@ -0,0 +1,30 @@
+class StoreSupplier: 
+    def __init__(self, products, sell_prices, email_address, phone_number, 
+        distance_from_shop, supply_capacity):
+        self.products = products                      #  needs to be a list 
+        self.sell_prices = sell_prices                #  needs to be a list 
+        self.email_address = email_address            #  needs to be a string
+        self.phone_number = phone_number              #  needs to be an integer
+        self.distance_from_shop = distance_from_shop  #  needs to be an integer
+        self.supply_capacity = supply_capacity        #  needs to be an integer
+        
+    def s_t_store():
```

Comment:
s_t*?

Reply:
Sorry, it was suposed to be an abbreviation. I'll modify the code right away.